game/time_to_run.py

```diff
@@ -1,14 +1,18 @@
 import time
 from copy import copy
 
+class TimeLimitReached(Exception):
+    def __init___(self, time_limit):
+        self.time_limit = time_limit
+
 class TimeLimitsCalculator(object):
     def __init__(self):
         self.prepare_time_limit = None
         self.play_time_limit = None
 
     def run(self):
         self.prepare_time_limit = self.time_to_complete(35)
```

Comment:
Sorry I didn't made it explicit, but based on what we defined in the rules, these should be actual fixed numbers (5 secs and 500 msecs) instead of approximations.